code/Qt/TrainWindow.cpp

```diff
@@ -83,8 +90,18 @@ void TrainWindow::trainingDataLoop(){
 TrainWindow::~TrainWindow(){
 }
 
-void TrainWindow::closeWindow(){
-
+void TrainWindow::startNeuralNet(){
+    
+    QList<QListWidgetItem *> selected_movements = movementList->selectedItems();
+    
+    std::string command = "python3 neuralNet.py ";
+    
+    for(int i=0; i<selected_movements.size(); i++){
+        command += (selected_movements[i]->text()).toStdString();
+        command += " ";
+    }
+    
+    system(command.c_str()); 
```

Comment:
starting a python enviroment within c might be better?
[Embedding Python in C++](https://www.codeproject.com/Articles/820116/Embedding-Python-program-in-a-C-Cplusplus-code)

Reply:
yeah I agree, just couldn't get to work and want to test it quickly, but I will change it.
I think the advantage is that QT will know when the script is finished